Build/Build.cs

```diff
@@ -178,6 +194,8 @@ class Build : NukeBuild
                 .SetProject(Solution.Core.FluentAssertions)
                 .SetOutputDirectory(ArtifactsDirectory)
                 .SetConfiguration("Release")
+                .EnableNoLogo()
+                .EnableNoRestore()
```

Comment:
There was a reason I had to keep restoring the dependencies, but I can't remember why.